Sources/Testing/Parameterization/CustomTestArgumentEncodable.swift

```diff
@@ -150,7 +150,7 @@ extension Encoder {
   ///
   /// The value of this property is non-`nil` when this encoder is being used to
   /// encode an argument passed to a parameterized test function.
-  @_spi(ExperimentalParameterizedTesting)
+  @_spi(ForToolsIntegrationOnly)
```

Comment:
This one actually should be `@_spi(Experimental)`, now. It's something which is intended to be evaluated for future promotion to API, and is meant for use by test authors, not so much by tools integrators

Reply:
Oops! Thanks for catching that. Will revise.